packages/sanity/src/_internal/cli/actions/deploy/helpers.ts

```diff
@@ -75,38 +88,86 @@ export async function getUserApplication({
 
 export async function getUserApplications({
   client,
+  organizationId,
 }: GetUserApplicationsOptions): Promise<UserApplication[] | null> {
+  const query = organizationId ? {organizationId: organizationId, appType: 'coreApp'} : undefined
```

Comment:
We should start making sure we send the appType always

```suggestion
  const query = organizationId ? {organizationId: organizationId, appType: 'coreApp'} : { appType: 'studio'}
```